style.css

```diff
@@ -0,0 +1,144 @@
+body {
+    margin: 0;
+    padding: 0;
+    }
+
+.main-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100vh;
+}
+
+.title-flex-container {
+    display: flex;
+    flex-wrap: nowrap;
+    align-content: center;
+    justify-content: center;
+    background: red;
+    border: solid white;
+}
+
+#yelpington-flex {
+    display: inline-block;
+    font-family: verdana;
+    font-size: 5vw;
+    background-color: red;
+    color: white;
+    border: solid;
+}
+
+.sidebar-map-container {
+    display: grid;
+    grid-template-rows: 25rem;
+    grid-template-columns: 1fr 5fr;
+    grid-template-areas: 
+    'sidebar-container map-container';
+}
+
+.sidebar-container {
+    flex: 1;
+    padding: 0 10px 0 10px;
+    border: 1px solid white;
+    border-radius: 10px;
+    background-image: linear-gradient(rgb(236, 116, 116), rgb(136, 29, 29));
+    color: red;
+}
+
+#restaurant-title {
+    text-align: center;
+    color: white;
+}
+
+.restaurants {
+    font-size: 17px;
+    list-style-type: none;
+}
+
+#restaurant-list {
+    background-color: (rgb(236, 116, 116), rgb(136, 29, 29));;
```

Comment:
Looks like you have an extra semicolon here, I think everything on your page is rendering how you intend it to, but be careful, sometimes extra characters like this will break your css